Scoop/Controller/Onboarding/AboutYouViewController.swift

```diff
@@ -47,78 +47,115 @@ class AboutYouViewController: OnboardingViewController {
             self.delegate?.didTapNext(navCtrl, nextViewController: nil)
         }
         
+        setupTitleLines()
         setupStackView()
+        backButton.isHidden = true
         setupNextButton(action: nextAction ?? UIAction(handler: { _ in
             return
         }))
+        
     }
     
     private func setupStackView() {
+        let stackviewMultiplier = 0.10
+        let leadingTrailingInset = 20.0
+        let spacing = 12.0
+        let screenSize = UIScreen.main.bounds
+        let labelFont = UIFont(name: "Rambla-Regular", size: 16)
+        let textFieldBorderWidth = 1.0
+        let textFieldCornerRadius = 8.0
+        let textFieldBorderColor = UIColor(red: 0.584, green: 0.616, blue: 0.647, alpha: 1).cgColor
```

Comment:
If these colors are reused elsewhere, for example in Eatery the `eatery blue` color is reused a lot, you might want to consider setting up an extension to UIColor and access these color as a static property